modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/yaml/YAMLUtil.java

```diff
@@ -82,7 +87,7 @@ public static OpenAPIYAML loadOpenAPIYAML(String yamlString) {
 
 		configYAMLConstructor.addTypeDescription(securityTypeDescription);
 
-		Representer representer = new Representer() {
+		Representer representer = new Representer(new DumperOptions()) {
```

Comment:
@jeffwu0724 Why this one is using a different one than the other two?

Reply:
Hi @tinatian .

Based on the documentation, we need to have DumperOptionas as parameter while declaring Representer, please see https://bitbucket.org/snakeyaml/snakeyaml/commits/094691f9392fbf6997d05aeaa4c615a434b41120

And we need to have LoaderOptions while declaring new Construcor as parameter , please see https://bitbucket.org/snakeyaml/snakeyaml/commits/68d199a0ef93fbc02639f891d06ea994b6134a0e.

I will include these two infos in my commit message.

Reply:
@jeffwu0724 Then please separate the changes into two commits.